### PR TITLE
Implement close button for tabs.

### PR DESCRIPTION
**Pull request by Gozala**
Implementation for #85

### CODE REVIEW

File: css/tabstrip.css [hunk 1]
```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
 .tab.selected .tab-thumbnail {
   transform: scale(1);
 }
+
+.tab-close-button {
+  font-family: FontAwesome;
+  text-align: center;
+  font-size: 16px;
+  vertical-align: middle;
+  width: 22px;
```
Comment (gordonbrander) [inline css/tabstrip.css]: Should be `width: 18px`, `height: 18px`

File: css/tabstrip.css [hunk 2]
```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
 .tab.selected .tab-thumbnail {
   transform: scale(1);
 }
+
+.tab-close-button {
+  font-family: FontAwesome;
+  text-align: center;
+  font-size: 16px;
+  vertical-align: middle;
```
Comment (gordonbrander) [inline css/tabstrip.css]: This doesn't do anything. You want `line-height: 18px`.

File: css/tabstrip.css [hunk 3]
```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
...
+  width: 22px;
+  height: 22px;
+  position: absolute;
+  top: 10px;
+  right: -10px;
```
Comment (gordonbrander) [inline css/tabstrip.css]: `right: -9px`

File: css/tabstrip.css [hunk 4]
```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
...
+
+.tab-close-button {
+  font-family: FontAwesome;
+  text-align: center;
+  font-size: 16px;
+  vertical-align: middle;
+  width: 22px;
+  height: 22px;
+  position: absolute;
+  top: 10px;
```
Comment (gordonbrander) [inline css/tabstrip.css]: `top: 7px`

File: css/tabstrip.css [hunk 5]
```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
...
+  display: none;
+}
+
+
+.tab-close-button:after {
+  content: "\f00d";
+}
+
+.tab:hover .tab-close-button {
+  display: inline-block;
```
Comment (gordonbrander) [inline css/tabstrip.css]: Nit: `display: block`. This is `position: absolute`, so you can't really use the fancier `display` models.

File: css/tabstrip.css (hunk 3 above)
Comment (paulrouget) [inline css/tabstrip.css]: It's a square, vertically and horizontally aligned. I would expect `top = -right`.

File: css/tabstrip.css [hunk 6]
```diff
@@ -77,3 +78,31 @@ main.showtabstrip iframes {
...
+  width: 22px;
+  height: 22px;
+  position: absolute;
+  top: 10px;
+  right: -10px;
+  background: #222;
+  color: white;
+
+  z-index: 2;
+  border-radius: 100%;
```
Comment (paulrouget) [inline css/tabstrip.css]: It's valid. But we usually use 50%.

**Pull request merged**